engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/job/preparation/JobPreparationActor.scala

```diff
@@ -236,7 +237,21 @@ class JobPreparationActor(workflowDescriptor: EngineWorkflowDescriptor,
                                                     maybeCallCachingEligible: MaybeCallCachingEligible,
                                                     prefetchedJobStoreEntries: Map[String, KvResponse],
                                                     dockerSize: Option[DockerSize]): BackendJobPreparationSucceeded = {
-    val jobDescriptor = BackendJobDescriptor(workflowDescriptor.backendDescriptor, jobKey, runtimeAttributes, inputEvaluation, maybeCallCachingEligible, dockerSize, prefetchedJobStoreEntries)
+    val memoryMultiplier: Option[MemoryRetryMultiplierRefined] = prefetchedJobStoreEntries.get(MemoryMultiplierKey) flatMap {
+      case KvPair(_,v) => refineV[MemoryRetryMultiplier](v.toDouble) match {
+        case Left(e) =>
+          // should not happen as we are converting a value that Cromwell put in DB after validation
+          log.error(e, s"Programmer error: unexpected failure attempting to read value for MemoryMultiplierKey from Key Value table. " +
+            "Expected value should be in range 1.0 ≤ n ≤ 99.0")
```

Comment:
Same comment as above re applying the limit to the factor vs to the multiplier

Reply:
Updated the error message. As above, having `Expected value should be in range 1.0 ≤ n ≤ 99.0` here was a copy/paste error 🤦‍♀️